include/cantera/zeroD/DeviceConst.h

```diff
@@ -0,0 +1,37 @@
+//! @file DeviceConst.h
+
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at http://www.cantera.org/license.txt for license and copyright information.
```

Comment:
Can you please use https://cantera.org/license.txt for all new files? Thank you :smile:

Reply:
And update the URL in any other files that you're changing, if it hasn't been done already.

Reply:
I can certainly do that ... I guess I copied the block from files that hadn't been updated.

Reply:
Done.